TestProject/cellularAutomat/Board.php

```diff
@@ -175,4 +184,79 @@ function checkBoardOnSimilarities(): bool
             return false;
     }
 
+    /**
+     * @param $_height
+     */
+    function setHeight($_height)
+    {
+        $this->height=$_height;
+    }
+
+    /**
+     * @return int
+     */
+    function getHeight(): int
+    {
+        return $this->height;
+    }
+
+    /**
+     * @param $_width
+     */
+    function setWidth($_width)
+    {
+        $this->width=$_width;
+    }
+
+    /**
+     * @return int
+     */
+    function getWidth(): int
+    {
+        return $this->width;
+    }
+
+    /**
+     * @param $_board
+     */
+    function setBoard($_board)
```

Comment:
Is this really in use? The board should probably not be directly accessible from the outside. What if someone overwrites the board with an array that has not the correct height/width?